docs/guide/installation.md

```diff
@@ -26,13 +26,13 @@ Apollo Boost is a zero-config way to start using Apollo Client. It includes some
 
 Install it alongside `vue-apollo` and `graphql`: 
 
-```
-npm install --save vue-apollo graphql apollo-boost
+```sh
+npm i vue-apollo graphql apollo-boost
```

Comment:
I think it's better to use the full word `install` in documentation. It's more explicit and understandable for inexperienced users. (same below)